doc/api/readline.md

```diff
@@ -90,7 +90,7 @@ The `'pause'` event is emitted when one of the following occur:
 
 * The `input` stream is paused.
 * The `input` stream is not paused and receives the `SIGCONT` event. (See
-  events `SIGTSTP` and `SIGCONT`)
+  events [SIGTSTP][] and [SIGCONT][])
```

Comment:
@addaleax as a reader, I do not know when I'm reading the `pause event` where the other events are placed, but for some reason, the documentation tells me that they are related, I feel like we can improve the **UX** adding links instead ``. We are just adding a shortcut.